.github/ISSUE_TEMPLATE/epic_issue.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Epic
+about: An organization level issue that progresses us towards our mission
+title: ''
+labels: epic
+assignees: ''
+
+---
+
+# Epic
+
+## Summary
+
+*Provide a summary of what this epic aims to accomplish or achieve.*
+
+## Tasks
```

Comment:
In my view, epics would be made up of "projects" and projects made up of tasks.

Reply:
Fair point @kieranhall. Can a project be 1 or more tasks or should they always have multiple tasks attached to them? If so, we'd quickly get weird mappings where an issue == project. As I see a task == issue.

I usually see an epic as a big overarching issue with multiple tasks/issues attached to them, but not necessarily as consisting of projects themselves.

For example, I wouldn't define `mainnet` as an epic, with `piecrust` and the `node` being a project with each their own tasks. 

I think it is also a bit against the way SCRUM defines an epic, as it is seen as a larger user story that encompasses a broader scope.

Should we get in a couple of others to define how we use an `epic` in Dusk?

Reply:
When I say "projects", I literally mean a [Github Project](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).  I would choose to have "epics" be a special kind of issue that is not tracked under any project, but instead, the epic is made into a label `Epic: Xyz` which can be added to any "Story" issue.  (I wouldn't suggest applying the epic label to every "Sub-Task" issue, though).

I'm aware that Epics are defined differently in SCRUM and that might potentially raise the use of "epic" as an issue type.  It would seem that we're using epics as organisational-level (high level) goals that fall under one of our (roadmap) milestones; whereas SCRUM tightly couples with the User Stories it contains.

Reply:
> When I say "projects", I literally mean a [Github Project](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects). I would choose to have "epics" be a special kind of issue that is not tracked under any project, but instead, the epic is made into a label `Epic: Xyz` which can be added to any "Story" issue. (I wouldn't suggest applying the epic label to every "Sub-Task" issue, though).
> 
> I'm aware that Epics are defined differently in SCRUM and that might potentially raise the use of "epic" as an issue type. It would seem that we're using epics as organisational-level (high level) goals that fall under one of our (roadmap) milestones; whereas SCRUM tightly couples with the User Stories it contains.

This differentiation is a level above what this issue would deal with in my view. This issue template, to me, is an overarching task containing sub tasks. 

What you describe would, in my view, fit more in a strategic overview of epics and milestones we have to achieve, like a roadmap.

We should probably push some more people into this discussion so we can finalize it and decide on what approach we should take here.